src/main/frontend/components/block.css

```diff
@@ -657,4 +657,52 @@ html.is-mac {
       cursor: pointer;
     }
   }
-}
\ No newline at end of file
+}
+
+/**
+ * Fix broken styles of breadcrumb caused by mixed contents
+ * TODO: implement the logic in code instead of CSS selector
+*/
+
+/*Fix images*/
+.breadcrumb .image-resize {
+  width: auto;
+  height: 17px;
+  display: inline-block;
+  vertical-align: middle;
+  top: -1px;
+}
+
+.breadcrumb .image-resize img {
+  height: 17px;
+  width: auto;
+}
+
+.breadcrumb .image-resize :is(.asset-action-bar, .erd_scroll_detection_container) {
+  display: none;
+}
+
+/*Fix iframe(e.g YouTube)*/
+.breadcrumb :is(.embed-block>div, iframe) {
+  display: none;
+}
+
+/*Fix embed-block, properties*/
+.breadcrumb>a>div:not([style="display: inline;"]),
+.breadcrumb .embed-block {
+  display: inline-block;
+}
+
+/*Use ... to replace the invalid items*/
+.breadcrumb .embed-block {
+  background: initial;
+}
+
+:is(.breadcrumb .embed-block, .breadcrumb>a>div:not([style="display: inline;"]))::after {
+  content: "...";
+}
+
```

Comment:
```css
/*fix property*/
.breadcrumb>a>div>.is-paragraph {
display:none;
}

/*fix twitter*/
.breadcrumb>a>span>div>div.contents::after {
content:"...";
} 
.breadcrumb .twitter-tweet,
.breadcrumb>a>span>div>span.flex.items-center {
display:none !important;
}
 .breadcrumb>a>span>div {
display:inherit;
}
```

Reply:
Not going to include. Using `!important` is not sustainable

Reply:
Should be solved in codebase instead of CSS